### PR TITLE
Feature one

### PR DESCRIPTION
**Pull request by addisalemseifu**
## Weather-App

**In this milestone I:**

- Created a React App.

- Set up the repository on GitHub and use Gitflow.

- Set up React and Redux.

- Set up testing libraries(React Testing Library and Jest).

- Installed React Redux, Redux Logger, and React Router.

- Created a dynamic route and viewed the Details page.

- Created directories for all Redux state slice files.

- Fetched data upon the first render from the Weather API endpoints:
  - http://api.weatherapi.com/v1/forecast.json?
- Used useSelector() Redux Hook to select the state slices and render lists of weather forecasts for the corresponding dates.

- Styled the whole project with plain CSS.

- Created Readme. md file.
fff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,186 @@
...
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<a name="readme-top"></a>
 
-## Available Scripts
+<div align="center">
+    <br/>
 
-In the project directory, you can run:
+ # Space-Traveler
```
Comment (DeliceLydia) [inline README.md]: - [ ] Change the name to the proper name of the project

File: README.md [hunk 2]
```diff
@@ -1,70 +1,186 @@
...
-### `npm run build`
+<!-- PROJECT DESCRIPTION -->
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+# 📖 [Space-Traveler] <a name="about-project"></a>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+"Weather-App" is a web application that allows users to search any city in the world and automatically find out all the weather information about that specific city. The appliction is also capable of displaying a weather prdiction for the comming 3 days. In addition to that the user can also get a detailed weather information for the hall 24 hours of the day.
```
Comment (DeliceLydia) [inline README.md]: - [ ] Change the name to the name of the project

File: README.md [hunk 3]
```diff
@@ -1,70 +1,186 @@
...
+If you like this project give it a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to acknowledge and give credit to Microverse and all the Microverse community.
+
```
Comment (DeliceLydia) [inline README.md]: - [ ] Please give proper credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Comment (DeliceLydia) [reply README.md]: - [ ] Please record a video 
- Presenting the features of the project
- Highlight a piece of the code you find interesting or proud of making

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @addisalemseifu, 

Good job so far! There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @addisalemseifu 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Correct use of react-router-dom ✔️ 
- Correct us redux store to store retrieved API ✔️ 
- Well-structured files :heavy_check_mark: 


Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**